apps/system/index.html

```diff
@@ -344,6 +346,8 @@
       <!-- XXX: Before we can make LockScreen as a iframe or app, we need this to split LockScreen and System part in this way (Bug 1057198) -->
       <div id="lockscreen-frame-placeholder"></div>
 
+      <div id="fullscreen-overlay" data-z-index-level="fullscreen-overlay"></div>
+
```

Comment:
The thing is that this dialog is a little bit different. It needs to be in top of the statusbar, but not covering the SHB. Not other overlay has this properties. I tried to put it in several places, but didn't find a way to accommodate the needs of this dialog. So this div would serve as a container to any other dialog with this properties (in the same way the dialog-overlay works)